packages/flutter/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## [3.1.3](https://github.com/parse-community/Parse-SDK-Flutter/compare/V3.1.2...flutter-3.1.3) (2022-06-06)
+
+### Bug Fixes
+
+* old version of `connectivity_plus package` ([#717](https://github.com/parse-community/Parse-SDK-Flutter/issues/717))
+* dependency `package_info_plus` does not work in web ([#714](https://github.com/parse-community/Parse-SDK-Flutter/issues/714))
+* missing plugin exception, no implementation found for method `getAll` ([#712](https://github.com/parse-community/Parse-SDK-Flutter/issues/712))
+
+### BREAKING CHANGES
```

Comment:
remove the "BREAKING CHANGES" section incl. the "upgrade various dependencies" entry, since we determined that this is irrelevant for the developer

Reply:
ok

Reply:
Done